### ISSUE TITLE
Favor exact match

### ISSUE THREAD

**Issue by jlecour** [topfunky/PeepOpen-Issues#277]
Hi,

I use PeepOpen a lot and I'm loving it, but something is annoying me everyday.

In my Rails projects, I have both Gemfile and Gemfile.lock files (from Bundler).
When I search for "Gemfile", the first item in the search results is always Gemfile.lock even if there is an exact match in the found files.

Thans for making PeepOpen even more awesome.

**Comment by topfunky** [comment]
Good point. I'll experiment with this and come up with a solution.

**Comment by jlecour** [comment]
Hi,

I've just notice that it's a matter of meta-data. It favors the last updated file where it should — imho — favor the best match.

If I `touch Gemfile`, then reload Peepopen and search for `gemfile` I get the correct file (`Gemfile`), but if I update the `Gemfile.lock` (via Bundler or `touch`) and reload Peepopen, I get the `Gemfile.lock` file first.

I don't know if I'm clear :-)

**Comment by topfunky** [comment]
Good point. It does prefer showing the last updated file. I'll experiment with this and see if exact match works better.
